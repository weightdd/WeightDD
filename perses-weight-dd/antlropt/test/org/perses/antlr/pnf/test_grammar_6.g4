grammar test_grammar_6;

enumerator: typedefName | typedefName '=' constantExpression ;