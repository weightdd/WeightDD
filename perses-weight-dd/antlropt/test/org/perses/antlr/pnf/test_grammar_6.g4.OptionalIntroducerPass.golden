// Auto-generated by Perses AST.
// DO NOT MODIFY.
grammar test_grammar_6;



enumerator
    : typedefName optional__enumerator_2
    ;

aux_rule__enumerator_1
    : '=' constantExpression
    ;

optional__enumerator_2
    : aux_rule__enumerator_1?
    ;

